### PR TITLE
AddNewLHComponent element made with example bundle files and go file

### PR DESCRIPTION
**Pull request by jarpinosynthace**

### CODE REVIEW

**Comment by chr1sgrant** [comment]
@synthacebot test this please

File: an/Utility/AddNewLHComponent.an [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+// The core process for this protocol. These steps are executed for each input.
+Steps {
+	
+	//Initialise variable err with type error 
+	var err error
+
+	//Store the TemplateComponent name into a variable for text output at the end of the run
+	TemplateComponentName := TemplateComponent.CName
```
Comment (chr1sgrant) [inline an/Utility/AddNewLHComponent.an]: Should this be exported?

**Comment by jarpinosynthace** [comment]
@chr1sgrant I have changed the code slightly, i had made a mistake. As it was it would have exported the TemplateLHComponent name which we dont really need. We need to export the NewLHComponent name. The export comment i had meant to say that it was saving the template name as a variabkle to be used in the Status output. I have updated the code so this is now not capitalised and that we now actually output in Data the NewLHComponentName

**Comment by chr1sgrant** [comment]
@synthacebot test this please

**Comment by chr1sgrant** [comment]
@synthacebot test this please

**Pull request merged**